src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -559,6 +560,11 @@ public static boolean onLivingAttack(EntityLivingBase entity, DamageSource src,
         return !MinecraftForge.EVENT_BUS.post(new LivingAttackEvent(entity, src, amount));
     }
 
+    public static boolean onLivingKnockBack(EntityLivingBase target, Entity attacker, float strength, double ratioX, double ratioZ)
+    {
+    	return !MinecraftForge.EVENT_BUS.post(new LivingKnockBackEvent(target, attacker, strength, ratioX, ratioZ));
```

Comment:
As you are negating the result when using this function, it'd be better to avoid the double-negation by just returning the result here directly.